x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.d.ts

```diff
@@ -23,3 +23,24 @@ export function hitsToGeoJson(
   geoFieldType: ES_GEO_FIELD_TYPE,
   epochMillisFields: string[]
 ): FeatureCollection;
+
+export function createExtentFilter(
+  mapExtent: MapExtent,
+  geoFieldName: string,
+  geoFieldType: ES_GEO_FIELD_TYPE
+): unknown;
+
+export function makeESBbox({
+  maxLat,
+  maxLon,
+  minLat,
+  minLon,
+}: {
+  maxLat: number;
+  maxLon: number;
+  minLat: number;
+  minLon: number;
+}): {
+  top_left: number[];
+  bottom_right: number[];
+};
```

Comment:
It might be worth creating a couple more types to remove that unknown on line 31 and add some return types to existing functions. Something like:
```
export type ESBBox = {
  top_left: number[];
  bottom_right: number[];
}

export type EnvelopeAsPolygon = {
  type: string,
  coordinates: Array<Array<number>>
};

export function createExtentFilter(
  mapExtent: MapExtent,
  geoFieldName: string,
  geoFieldType: ES_GEO_FIELD_TYPE
): Record<string, Record<string, {
  shape: EnvelopeAsPolygon,
  relation: string
} | ESBBox>>;

export function makeESBbox({
  maxLat,
  maxLon,
  minLat,
  minLon,
}: {
  maxLat: number;
  maxLon: number;
  minLat: number;
  minLon: number;
}): ESBBox;
```